syft/workers/base.py

```diff
@@ -439,6 +443,21 @@ def execute_command(self, message: tuple) -> PointerTensor:
                 new_ids = return_id_provider.get_recorded_ids()
                 raise ResponseSignatureError(new_ids)
 
+    def execute_plan_command(self, message: tuple):
+        """Executes commands related to plans.
+
+        This method is intended to execute all commands related to plans and
+        avoiding having several new message types specific to plans.
+
+        Args:
+            message: A tuple specifying the command and args.
+        """
+        command_name, args = message
+        if command_name in self._plan_command_router:
```

Comment:
How often do we expect to get a command not in the command router? This would probably be faster as a try except since I expect that in general we not have incorrect commands very often (or at all)

Reply:
The problem with try except is that if we get a `keyError` while running the command itself we'll get the wrong error message (this happened to me a few times and it was very confusing).

Reply:
So if we do 
```
try:
    code here
except KeyError as e:
    raise PlanCommandUnknownError(command name)
```

Reply:
Yes, but if <code here> throws a KeyError by another reason (for instance we try to get_obj for an unknown obj) we would raise the wrong error. I think this could be fixed on another PR where we create a get_obj exception. What do you think?

Reply:
I'll use try except for now, but we should probably raise a syft exception at get_obj https://github.com/OpenMined/PySyft/blob/750bd2c3987a570ef1dc8554b933696d416724c7/syft/generic/object_storage.py#L84

Reply:
Agreed

Reply:
I've created an issue for this: https://github.com/OpenMined/PySyft/issues/2608